front/js/components/successOrder.es6

```diff
@@ -0,0 +1,29 @@
+(() => {
+  const DOM = {
+    $purchasedOrder: $('.js-purchased-order'),
+  };
+
+  const init = () => {
+  	publishSuccessOrder();
+  };
+
+  function publishSuccessOrder() {
+    if (!DOM.$purchasedOrder.length) {
```

Comment:
let's move the whole if to the separated local function like `pageContainsOrderData`